pkgs/tools/misc/fwup/default.nix

```diff
@@ -1,26 +1,49 @@
-{ stdenv, lib, fetchFromGitHub, autoreconfHook, makeWrapper, pkg-config
-, zlib, lzma, bzip2, mtools, dosfstools, zip, unzip, libconfuse, libsodium
-, libarchive, darwin, coreutils }:
+{ lib
+, autoreconfHook
+, bzip2
+, coreutils
+, darwin
+, dosfstools
+, fetchFromGitHub
+, libarchive
+, libconfuse
+, libsodium
+, lzma
+, makeWrapper
+, mtools
+, pkg-config
+, stdenv
+, unzip
+, which
+, xdelta
+, zip
+, zlib
+}:
```

Comment:
Inputs are typically sorted first by group: general use in derivation (`stdenv`, `lib`, fetchers, etc.), `nativeBuildInputs`, `buildInputs`, and then `checkInputs`. Then the inputs are sorted alphabetically per group.

I don't think it is a blocker, but IMO it's preferable to keep that order.

Reply:
Guess we are looking at `isort`-style sorting here. I can arrange the inputs in that way.